pkg/config/config.go

```diff
@@ -412,6 +412,7 @@ type (
 		VerifyJWT  *bool  `toml:"verify_jwt" json:"verifyJWT"`
 		ImportMap  string `toml:"import_map" json:"importMapPath,omitempty"`
 		Entrypoint string `json:"-"`
+		NoDeploy   bool   `toml:"no_deploy"`
```

Comment:
```suggestion
		Enabled   bool   `toml:"enabled"`
```

We can use `enabled` option for consistency with rest of the config.